### PR TITLE
updated index.rst for time format customization

### PR DESCRIPTION
**Pull request by mandarGG**
issue #4631 resolved

### CODE REVIEW

**Comment by eteq** [comment]
The tests are failing here because the text added does not correspond to the Sphinx format for code blocks.  I suggest you look at the other code examples in the same file, and perhaps also take a look at the sphinx and in particular the [ReST primer](http://www.sphinx-doc.org/en/stable/rest.html). 

In particular one of the problems here is you don't have `>>>` in front of the actual python code.

It would also be good to hear from @taldcroft whether he thinks this should be at the *top* of this section - my hunch is that it's supposed to be at the bottom of the "Time format" section

File: docs/time/index.rst [hunk 1]
```diff
@@ -174,6 +174,10 @@ yday         :class:`~astropy.time.TimeYearDayTime`             2000:001:00:00:0
    initialization.  Furthermore, any specific realization information,
    such as ``UT(NIST)`` is stored only as long as the time scale is not changed.
 .. [#] `Rots et al. 2015, A&A 574:A36 <http://adsabs.harvard.edu/abs/2015A%26A...574A..36R>`_
+
```
Comment (pllim) [inline docs/time/index.rst]: Are 4 blank lines necessary? Perhaps 2 are sufficient?

Comment (taldcroft) [inline docs/time/index.rst]: In general one should not change code or text that is unrelated to your new content.  So back out this change to add 4 lines here.

**Comment by taldcroft** [comment]
@mandarGG  - this should go into the http://astropy.readthedocs.org/en/latest/time/index.html#writing-a-custom-format section.  Also, the point is that we want to show the code example which *implements* the custom class, not just using a custom class.

**Comment by pllim** [comment]
Closing this in favor of #4902. Thanks for the effort anyway!
